src/main/resources/zenodo-1.0.0-swagger-2.0.yaml

```diff
@@ -12,7 +12,7 @@ basePath: /api
 schemes:
   - https
 paths:
-  /communities/:
```

Comment:
See https://github.com/dockstore/zenodo-client/pull/27/commits/4c91320cb64ca50ce73d708b8ca8fd0a443260e1 and